c7n/offhours.py

```diff
@@ -206,23 +210,37 @@ def __call__(self, i):
         return self.process_current_time(i, parts)
 
     def process_current_time(self, i, parts):
-        tz = self.get_local_tz(parts)
+        try:
+            parsed = ScheduleParser().parse(parts[0])
+            if parsed is None:
+                raise ValueError("malformed downtime tag")
+        except Exception as e:
+            log.warning("downtime tag is malformed or empty, using defaults")
```

Comment:
ideally this should log the value that was non parseable, its already size bound per tag value constraints.